src/runner/trace.cpp

```diff
@@ -18,13 +20,45 @@ void Trace::UnregisterProvider()
     TraceLoggingUnregister(g_hProvider);
 }
 
-void Trace::EventLaunch(const std::wstring& versionNumber)
+void Trace::EventLaunch(const std::wstring& versionNumber, bool isProcessElevated)
 {
     TraceLoggingWrite(
         g_hProvider,
         "Runner_Launch",
         TraceLoggingWideString(versionNumber.c_str(), "Version"),
         ProjectTelemetryPrivacyDataTag(ProjectTelemetryTag_ProductAndServicePerformance),
         TraceLoggingBoolean(TRUE, "UTCReplace_AppSessionGuid"),
-        TraceLoggingKeyword(PROJECT_KEYWORD_MEASURE));
+        TraceLoggingKeyword(PROJECT_KEYWORD_MEASURE),
+        TraceLoggingBoolean(isProcessElevated, "Elevated"));
```

Comment:
Please, move `TraceLoggingBoolean(isProcessElevated, "Elevated")` after `TraceLoggingWideString(versionNumber.c_str(), "Version")`, it's not functional just I prefer that all the application event data is in one place.